src/functionaltests/java/com/ericsson/ei/utils/DataBaseManager.java

```diff
@@ -127,6 +127,7 @@ public boolean verifyAggregatedObjectExistsInDB() throws InterruptedException {
         long stopTime = System.currentTimeMillis() + 30000 + extraCheckDelay;
         while (!eventsIdList.isEmpty() && stopTime > System.currentTimeMillis()) {
             eventsIdList = compareSentEventsWithEventsInDB(eventsIdList);
+            System.out.println("******eventsIdList********"+eventsIdList.size());
```

Comment:
Remove these when the tests are working again.

Reply:
removed.